lib/gui/components/safe-webview.js

```diff
@@ -198,26 +199,29 @@ class SafeWebview extends react.PureComponent {
    * @param {Event} event - event object
    *
    * @example
+   *
+   * // In the webview
    * console.log({
-   *   type: 0,
-   *   message: 'Hello, World!'
+   *   command: 'error',
+   *   data: 'Good night!'
    * });
+   *
+   * // or with robot
+   * robot.printError('Good night!');
```

Comment:
I don't think we should (for security reasons, it makes sense to keep the webview as isolated as possible). And therefore this `robot.printError` example probably isn't useful?